assets/app.js

```diff
@@ -318,6 +318,15 @@ new Vue({
             })
             return stash
         },
+        activeLanguagesId: function () {
+             return this.supportedLanguages.map(function (item) {
+                if (item.status){
+                   return item.id
+                }
+            }).filter(function (item) {
```

Comment:
For what you need to use `.filter()`? If `item` can be undefined, why not just `if( item !== undefined && item.status)` ? And are you sure item can be undefined?

Reply:
I use `.filter()` because when I use ` .map()`  if `item.status` is `false` it will return `undefined`  that make  `activeLanguagesId` have `undefined`  data inside.

but ok, if it doesn't make sense to use `map` and `filter` I will use `foreach` instead.